app/assets/stylesheets/search.scss

```diff
@@ -137,7 +137,9 @@
 .search-usersearch-image {
   display: block;
   margin-bottom: 20px;
-  width: 70%;
+  width: 150px;
+  border-radius: 8px;
+  height: 150px;
```

Comment:
You can keep `width: 70%` and use aspect-ratio and object-fit properties to keep it square irrespective of image.

Ref - 
https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio
https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit

Reply:
@tanmoysrt Done

Reply:
Can you tryout aspect-ratio and object-fit, it will keep the image in square shape as many user upload rectangular image and that doesn't looks good.

Reply:
@tanmoysrt Added  aspect-ratio and object-fit.